universal-login-relayer/src/integration/sql/migrations/20200330000001_create_future_wallets.js

```diff
@@ -0,0 +1,21 @@
+/*
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
+  await knex.schema.createTable('future_wallets', (table) => {
+    table.increments();
+    table.string('contractAddress').notNullable();
+    table.string('publicKey').notNullable();
```

Comment:
What do you think about specifying length? (42)

Reply:
We can do it